pii_data.py

```diff
@@ -21,41 +30,42 @@ def has_email(self):
             return True
         return False
 
-    def has_ipv4(self):
+    def has_ipv4(self, anonymize=False):
         # Match an IPv4 address: num.num.num.num where num range = 0 - 255
 
         # [0-9]:        match numbers 0 - 9
```

Comment:
There is no need to change anything here, but, I see that you documented what your regex was doing - which is a very good best practice because they can be so complicated to decipher.  I wanted to point out the [re.VERBOSE](https://docs.python.org/3.9/library/re.html?highlight=subn#re.VERBOSE) option which allows you to document a regex embedded within the regex. Some people do it this way when the string is very complicated.

Reply:
Thanks, I read through the documentation and that will be something I will try to implement more